hack/config.sh

```diff
@@ -1,5 +1,5 @@
 binaries="cmd/virt-controller cmd/virt-launcher cmd/virt-handler cmd/virt-api cmd/virtctl"
-docker_images="$binaries images/haproxy images/iscsi-demo-target-tgtd images/vm-killer"
+docker_images="$binaries images/haproxy images/iscsi-demo-target-tgtd images/vm-killer images/libvirtd"
```

Comment:
We need to find a solution to build and publish libvirt without kubevirt. It takes ages to build that container (wasting our time on every CI test run and developer recreate) and it will almost never be changed. At least we should have a base image which has libvirt already installed. We then just install the kubevirt stuff on top.

We can do that later on, but I would vote for a switch right now in hack/config.sh, to either use libvirt from the host or from here, until we do separate releases. Or we tackle the separete push right now ... 

In principle the same applies for the iscsi image too.